src/startup/ModeCLIParser.cpp

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (c) 2019 DKFZ - ODCF
+ *
+ * Distributed under the MIT License (license terms are at https://github.com/dkfz-odcf/FastqIndEx/blob/master/LICENSE.txt).
+ */
+
+#include "process/io/PathSink.h"
+#include "process/io/PathSource.h"
+#include "process/io/s3/S3Sink.h"
+#include "process/io/s3/S3Source.h"
+#include "process/io/Source.h"
+#include "process/io/ConsoleSink.h"
+#include "process/io/StreamSource.h"
+#include "ModeCLIParser.h"
+#include <tclap/CmdLine.h>
+
+#include <memory>
+
+_IntValueArg ModeCLIParser::createVerbosityArg(CmdLine *cmdLineParser) const {
+    return _makeIntValueArg(
+            "v",
+            "verbosity",
+            string("Sets the verbosityArg of the application in the range of 0 (default, less) to 3 (debug, max). ") +
```

Comment:
```suggestion
            string("Sets the verbosity. In the range of 0 (default, less) to 3 (debug, max). ") +
```
This is CLI-user documentation, or not?